jade/containers.rst

```diff
@@ -5,7 +5,7 @@ Using Containerised Applications
 
 On entering the container the present working directory will be the user's home directory: ``/home_directory``
 
-Any files you copy into ``/home_directory`` will have the same userid as normal and will be available once exiting the container. The local disk space on the node is available at: ``/local_scratch/$USERID``
+Any files you copy into ``/home_directory`` will have the same userid as normal and will be available once exiting the container. The local disk space on the node is available at: ``/raid/local_scratch/$USERID``
```

Comment:
Got a response from Hartree, looks like when using docker container, the scratch directory are mounted directly to `/local_scratch` and no username/id needs to be appended.

```suggestion
Any files you copy into ``/home_directory`` will have the same userid as normal and will be available once exiting the container. The local disk space on the node is available at: ``/local_scratch/``
```